dev-example/src/example.test.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  createMockMachineRunner,
+  createMockState,
+  createMockStateOpaque,
+} from '@actyx/machine-runner/lib/test-utils'
+import { Bid, BidderID, Passenger, Requested } from './machines/index.js'
+import { toPrettyJSONString } from './UIMachineCommon.js'
+import { isTaxiRideCancelEnabled } from './UIMachinePassenger.js'
+
+describe('State transformation tests', () => {
+  it('should verify if state transformation as written by the consumer code is correct', () => {
+    const requestDestination = 'destination'
+    const requestPickup = 'pickup'
+    const bidTime = new Date()
+    const bidPrice = 1
+    const bidderId = 'bidderId'
+
+    const machineRunner = createMockMachineRunner(Passenger.Initial, void 0)
+    machineRunner.test.feed([
+      Requested.make({
+        destination: requestDestination,
+        pickup: requestPickup,
+      }),
+      Bid.make({
+        price: bidPrice,
+        time: bidTime.toISOString(),
+      }),
+      BidderID.make({
+        id: bidderId,
+      }),
+    ])
```

Comment:
This would be the testing principle for event transitions — I assume that it also works when starting from a different state.

Reply:
It does